src/NUnitFramework/tests/Attributes/ParameterizedTestFixtureTests.cs

```diff
@@ -199,4 +201,56 @@ public void MakeSureTypeIsInSystemNamespace()
             Assert.That(_someType.Namespace, Is.EqualTo("System"));
         }
     }
+
+    [TestFixture("Zero")]
+    [TestFixture("One", 1)]
+    [TestFixture("Many", 1, 2, 3, 4)]
+    [TestFixture(1.5, 8.2)]
+    [TestFixtureSource(nameof(SourceData))]
+    public class ParameterizedTestFixtureWithParamsArgument
+    {
+#pragma warning disable NUnit1028 // The non-test method is public
+        public static IEnumerable SourceData()
+#pragma warning restore NUnit1028 // The non-test method is public
```

Comment:
Instead if suppressing the errors, make the `SourceData` method `private`.

Reply:
Oh didn't know that worked that's cool